pynestml/codegeneration/code_generator.py

```diff
@@ -42,18 +42,11 @@
 
 
 class CodeGenerator(WithOptions):
-    _default_options: Mapping[str, Any] = {}
-
-    def __init__(self, target, options: Optional[Mapping[str, Any]] = None):
-        from pynestml.frontend.pynestml_frontend import get_known_targets
+    r"""Generate code for a given target platform."""
 
-        if not target.upper() in get_known_targets():
-            code, msg = Messages.get_unknown_target(target)
-            Logger.log_message(message=msg, code=code, log_level=LoggingLevel.ERROR)
-            self._target = ""
-            raise InvalidTargetException()
+    _default_options: Mapping[str, Any] = {}
 
```

Comment:
Why is this removed?

Reply:
``self._target`` was apparently no longer being used, and specifying an invalid target is already prevented earlier in the frontend.